src/main/kotlin/lotto/usecase/LottoMachine.kt

```diff
@@ -1,33 +1,50 @@
 package lotto.usecase
 
 import lotto.domain.Lotto
+import lotto.domain.model.LottoNumber
+import lotto.domain.model.LottoNumbers
+import lotto.domain.model.Price
 
 class LottoMachine(
     private val lottoGenerator: LottoGenerator,
 ) {
 
-    fun buy(purchaseAmount: Int): List<Lotto> {
-        val numberOfPurchase = purchaseAmount / LOTTO_PRICE
+    fun automaticBuy(automaticCount: Int): List<Lotto> {
         val lottos = mutableListOf<Lotto>()
 
-        repeat(numberOfPurchase) {
-            val numbers = (MIN_NUMBER..MAX_NUMBER)
+        repeat(automaticCount) {
+            val lottoNumbers = (MIN_NUMBER..MAX_NUMBER)
                 .shuffled()
                 .subList(START_INDEX, END_INDEX)
                 .sorted()
-            val lotto = lottoGenerator.generate(numbers, LOTTO_PRICE)
+                .map { number -> LottoNumber(number) }
+            val lotto = lottoGenerator.generate(
+                numbers = LottoNumbers(lottoNumbers),
+                price = Price(LOTTO_PRICE)
+            )
 
             lottos.add(lotto)
         }
 
         return lottos.toList()
     }
 
+    fun passivityBuy(
+        passivityLottoNumbers: List<LottoNumbers>
+    ): List<Lotto> {
+        return passivityLottoNumbers.map { passivityLottoNumber ->
+            lottoGenerator.generate(
+                numbers = passivityLottoNumber,
+                price = Price(LOTTO_PRICE),
+            )
+        }
+    }
+
     companion object {
         private const val MIN_NUMBER = 1
         private const val MAX_NUMBER = 45
         private const val START_INDEX = 0
         private const val END_INDEX = 6
-        private const val LOTTO_PRICE = 1000
+        const val LOTTO_PRICE = 1000
```

Comment:
```suggestion
        const val LOTTO_PRICE = 1_000
```
천 단위 자리에 언더바를 붙여주시면 가독성이 좋아집니다

Reply:
완료했습니다.
**b4f15fb5**